src/phoenix/server/api/routers/v1/__init__.py

```diff
@@ -0,0 +1,11 @@
+from starlette.routing import Route
+
+from . import evaluations, spans, traces
+
+V1_ROUTES = [
+    Route("/v1/evaluations", evaluations.post_evaluation, methods=["POST"]),
+    Route("/v1/evaluations", evaluations.get_evaluations, methods=["GET"]),
+    Route("/v1/traces", traces.post_traces, methods=["POST"]),
+    Route("/v1/spans", spans.query_spans_handler, methods=["POST"]),
+    Route("/v1/spans", spans.get_spans_handler, methods=["GET"]),
```

Comment:
Is there no nested routing? NBD, just curious.

Reply:
I'm not sure everything we can accomplish here, this was the pattern I saw recommended to use Starlette's OpenAPI APISpec mechanism, since we want a separate function for each route